### PR TITLE
solution de principal vs secundary version 2

### PR DESCRIPTION
**Pull request by LuisGuardia**

### CODE REVIEW

**Comment by LuisGuardia** [comment]
hello, I created this new branch "luis/principal-vs-secondary-v2" because with the previous one I couldn't do the pull request.

File: src/principal-vs-secondary/luis/Matrices.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
+package matrices;
+
+/**
+ *
+ * @author: Luis Guardia
+ */
+public class Matrices {
```
Comment (J0c4) [inline src/principal-vs-secondary/luis/Matrices.java:8]: Naming needs to be changed to english

File: src/principal-vs-secondary/luis/Matrices.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    public Matrices(){
+        
+    }
+    
+    public void sumarMatriz(){
+        for (int x = 0; x < matriz.length; x++) {
+            diagonalPrincipal += matriz[x][x];
+            diagonalSecundaria += matriz[(matriz.length - 1) - x][x];
+        }
+        if(diagonalPrincipal > diagonalSecundaria)
```
Comment (J0c4) [inline src/principal-vs-secondary/luis/Matrices.java:22]: braces `{ }` missing

Comment (J0c4) [inline src/principal-vs-secondary/luis/Matrices.java:22]: space between `if` and `(` missing

File: src/principal-vs-secondary/luis/Matrices.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * @author: Luis Guardia
+ */
+public class Matrices {
+
+    int [][] matriz = {{2,4,4},{6,6,9},{18,10,12}};
+    int diagonalPrincipal,diagonalSecundaria;
+    
+    public Matrices(){
+
```
Comment (J0c4) [inline src/principal-vs-secondary/luis/Matrices.java:14]: I think this default constructor can be removed

**Pull request merged**